cmd/kubeadm/app/util/copy.go

```diff
@@ -17,19 +17,30 @@ limitations under the License.
 package util
 
 import (
+	"io"
 	"os"
 )
 
 // CopyFile copies a file from src to dest.
 func CopyFile(src, dest string) error {
```

Comment:
see https://github.com/kubernetes/kubernetes/pull/123254

recently we reverted a copy file unti test, would these changes break it again?

Reply:
we also need to make sure it works on windows.
i think all of these stdlib functions are portable to Windows, but hopefully there are no gotchas.